content/docs/authorization/cli.md

```diff
@@ -1095,3 +1096,42 @@ karavictl tenant delete [flags]
 $ karavictl tenant delete --name Alice
 ```
 On success, there will be no output. You may run `karavictl tenant get --name <tenant-name>` to confirm the deletion occurred.
+
+
+
+---
+
+
+
+### karavictl tenant update
+
+Updates a tenant's resource within CSM
+
+##### Synopsis
+
+Updates a tenant resource within CSM
+
+```
+karavictl tenant update [flags]
+```
+
+#### Options
+
+```
+  -h, --help   help for create
+  -n, --name string   Tenant name
+      --approvesdc boolean (Usage: --approvesdc=true/false | This flag is only applicable to PowerFlex. true value allows the tenant's SDC approval request to be forwarded to the PowerFlex array. false value will block the request to approve SDC if SDC is not in approved state)
```

Comment:
Since this flag is being specified as boolean then `Usage: --approvesdc=true/false` isn't necessary. Can you align Tenant name and This flag ... statements?

Can this be refined to say something like `Approve/Deny a tenant's SDC request `?

Reply:
It was kept as `--approvesdc=true/false` so as to not use the flag as `approvesdc value` as it is for other flags.

Reply:
Refined the statement as requested